### ISSUE TITLE
`_config` compilation ignores plain JS and CommonJS-style declarations

### ISSUE THREAD

**Issue by stevebest** [eHealthAfrica/couchdb-bootstrap#12]
Let's say I want to define slightly different configs for my dev and prod couches. From the docs it looks like this should work:

```js
// _config.js OR _config/index.js
module.exports = {
  "vhosts": (function (env) {
    return env === 'production'
       ? { "mydb.production-couch.com": "/mydb/_design/app/_rewrite" }
       : { "mydb.localhost:5984": "/mydb/_design/app/_rewrite" };
  })(process.env.NODE_ENV)
}
```

However, only `_config.json` seems to be recognized. Am I doing it wrong?

**Comment by jo** [comment]
That should work. Seems like a bug. Closing in favor of https://github.com/eHealthAfrica/couchdb-configure/issues/4

**Comment by jo** [comment]
Fixed in `v1.10.3`
